Apply Terra-ish look-and-feel

rsasch opened the pull request:
Job List:
![job-list](https://user-images.githubusercontent.com/1713505/54225986-376a1c80-44d3-11e9-9845-6e7ac0b171e4.png)


Job Details:
![job-details](https://user-images.githubusercontent.com/1713505/54160515-23b4ac80-4426-11e9-86f0-712807a7b76e.png)


Closes #361

rexwangcc commented:
I'm going to list some findings in the comments :)

rexwangcc commented:
When the Cromwell returns an error, the style seems to overlap with the error message
![Screen Shot 2019-03-11 at 10 38 28 PM](https://user-images.githubusercontent.com/14366016/54170827-7dc96800-444e-11e9-9f2a-70b3ba0066ae.png)

rexwangcc commented:
When you compress the window size, the top bar seems to shadow the text:
![Screen Shot 2019-03-11 at 10 43 14 PM](https://user-images.githubusercontent.com/14366016/54170982-34c5e380-444f-11e9-9cd1-c74a82b57099.png)

while the old one doesn't seem to have this issue:
![Screen Shot 2019-03-11 at 10 44 51 PM](https://user-images.githubusercontent.com/14366016/54171020-52934880-444f-11e9-9d7b-e46c3e75d074.png)

rexwangcc commented:
Somehow the text in the buttons on the list view page is not aligned to center on my local JMUI instance...
![Screen Shot 2019-03-11 at 10 45 30 PM](https://user-images.githubusercontent.com/14366016/54171053-735b9e00-444f-11e9-8558-963102691cda.png)

rexwangcc commented on ui/src/app/assets/images/header-left-hexs.svg:
```diff
@@ -0,0 +1,50 @@
+					<path class="st1" d="M60,7.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7.8
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C60.6,5.9,60,6.8,60,7.8"/>
+					<path class="st1" d="M36,7v8.4c0,1,0.6,1.9,1.5,2.4L45,22c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C36.6,5.1,36,6,36,7"/>
```
What does this do?

rsasch replied:
This is an image that is laid on top of the "Job Manager" header; see it rendered [here](https://github.com/DataBiosphere/job-manager/blob/7f059301678fa7c34ebb2d1f99ffe28cd397ef6e/ui/src/app/assets/images/header-left-hexs.svg ).

rexwangcc replied:
Wow, this is ..... awesome 😲

rsasch commented:
@rexwangcc I wasn't able to reproduce

> When you compress the window size, the top bar seems to shadow the text:

but I added some CSS to prevent the header cells text from wrapping to a new line.
I also wasn't able to reproduce

> Somehow the text in the buttons on the list view page is not aligned to center on my local JMUI instance...

but I tweaked the CSS to hopefully make that more stable.
And I fixed the header location on the error page.

rexwangcc commented:
@rsasch Thanks for the quick response!

I can see the header location on the error page is fixed now.

regarding the
> When you compress the window size, the top bar seems to shadow the text:

You could go to http://localhost:4200/jobs?q=status%3DOnHold and drag the edge of the Chrome window all the way left to make it slim, and the query builder bar will start to shadow the list view's headers.

rsasch commented:
@rexwangcc I guess I'm not clear on what the preferred behavior is when the screen is made that small.  Can you elaborate?

rexwangcc commented:
@rsasch I'm not sure what the preferred behavior is and having second thought now since I just found this issue on the current version of JMUI. It should be addressed in a standalone issue not this PR.

rsasch commented:
Thanks for the reviewing! 🥇
